docs/changelog/137818.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 137818
+summary: ES|QL Views REST API
+area: "ES|QL"
+type: feature
+issues: []
```

Comment:
For now this PR should be marked as `>non-issue` so that no changelog is generated. That way it does not show up in the public changelog until the feature flag has been removed (that one will have `>feature` and `>release-highlight` on it).